stages/org.osbuild.sysconfig

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python3
+"""
+
+Configure sysconfig files
+
+The sysconfig directory contains a variety of system configuration files. The
+values found in each of the configuration files may need to be specified
+before first boot.
+
+Currently, only the kernel and network configs can be modified.
+
+"""
+
+import sys
+import os
+
+import osbuild.api
+
+
+SCHEMA = """
+"additionalProperties": false,
+"properties": {
+  "kernel": {
+    "additionalProperties": false,
+    "type": "object",
+    "description": "sysconfig kernel options",
+    "properties": {
+      "update_default": {
+        "type": "boolean",
+        "description": "This option makes a newly installed kernel as the default in the boot entry selection."
+      },
+      "default_kernel": {
+        "type": "string",
+        "description": "This option specifies what package type will be used as the default."
+      }
+    }
+  },
+  "network": {
+    "additionalProperties": false,
+    "type": "object",
+    "description": "sysconfig network options",
+    "properties": {
+      "networking": {
+        "type": "boolean",
+        "description": "Enables or disables networking"
+      },
+      "no_zero_conf": {
+        "type": "boolean",
+        "description": "Disables the zero configuration network suite"
+      }
+    }
+  }
+}
+"""
+
+
+# sysconfig uses yes and no instead of true and false
+def bool_to_string(bool):
+    if bool:
+        return "yes"
+    else:
+        return "no"
+
+
+def configure_kernel(tree, kernel_options):
+    with open(f"{tree}/etc/sysconfig/kernel", 'w') as kernel_file:
```

Comment:
Both options overwrite the existing configuration, which is what anaconda does as well. We should at least document that. I wonder if we should also make sure the file is indeed missing/empty since overwriting existing configuration might be surprising in the future.